js/localgov_eu_cookie_compliance.es6.js

```diff
@@ -103,10 +103,24 @@
    * Provide feedback when the "Save cookie settings" button is clicked.
    */
   function setupSaveSettingsFeedback() {
-    $(".eu-cookie-compliance-save-preferences-button").click(e => {
-      $(`<p>${Drupal.t("Saved")}</p>`)
-        .appendTo(e.target)
-        .hide(2000);
-    });
+    var saveButton = document.querySelector('.eu-cookie-compliance-save-preferences-button');
+    const saveButtonLabel = saveButton.innerHTML;
+
+    saveButton.addEventListener("click", function () {
+      saveButton.innerHTML = `${Drupal.t("Saving")}`;
+      saveButton.classList.add('spinning');
+
+      setTimeout(
+        function () {
+          saveButton.classList.remove('spinning');
+          saveButton.innerHTML = `${Drupal.t("Preferences saved")}  <i class="fas fa-check"></i>`;
+        }, 2000);
+
+    }, false);
+
+    saveButton.addEventListener("blur", function () {
+      saveButton.innerHTML = saveButtonLabel;
+    }, false);
+
   }
 })(jQuery, Drupal);
```

Comment:
Is jQuery needed here? It would be good to not have a jQuery dependency in this module if possible.

Reply:
Agreed, but as this isn't a change in this PR I think we should open a new issue to drop the jQuery dep.

Reply:
Good idea. Can you create that issue please?

Reply:
I have raised a [ticket](https://github.com/localgovdrupal/localgov_eu_cookie_compliance/issues/6) to this effect.